src/components/RefinementList/__tests__/RefinementListItem-test.tsx

```diff
@@ -2,10 +2,27 @@
 
 import { h } from 'preact';
 import { shallow } from 'enzyme';
+import { ReactElementLike } from 'prop-types';
 import RefinementListItem from '../RefinementListItem';
 
+type Props = {
```

Comment:
if the props were exported from the component, you could use them here instead of repeating